interop-switch/values.yaml

```diff
@@ -43,7 +45,7 @@ service:
 
 ingress:
   enabled: true
-  externalPath: /switch/v1
+  externalPath: /interop/switch/v1
```

Comment:
Why the change?

Reply:
Initially started with /switch/v1. But with USSD integration realized that the scheme adapter uses /interop/switch/v1. To make the demo work without changes on USSD side, had to make this change. We can revert it back after the demo.